kotlin-js/src/jsMain/kotlin/js/objects/Object.kt

```diff
@@ -19,6 +19,7 @@ external class Object internal constructor() {
         fun <T : Any> entries(o: ReadonlyRecord<String, T>): ReadonlyArray<JsTuple2<String, T>>
         fun entries(o: Any): ReadonlyArray<JsTuple2<String, Any?>>
         fun <R, T : R> freeze(o: T): R
+        fun <T : Any> fromEntries(entries: ReadonlyArray<JsTuple2<String, T>>): Record<String, T>
```

Comment:
1. Bound for `T` is redundant
2. Return type `Record` -> `ReadonlyRecord`?
3. Could you please add overload with `JsIterable` as parameter?

Reply:
pushed